project/Vagrantfile

```diff
@@ -53,30 +63,31 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         end
         (ip = /^\s*inet .*?(\d+\.\d+\.\d+\.\d+)\s+/.match(result)) && ip[1]
     end
-    
+
     # Customize the virtual machine environments
     config.vm.provider :virtualbox do |vb|
         vb.customize ["modifyvm", :id, "--nictype1", "virtio"]
         vb.customize ["modifyvm", :id, "--nictype2", "virtio"]
         #vb.gui = true # for debugging
     end
-    
+
     config.vm.define mastername do |master|
         master.vm.box = "chef/centos-6.5"
         master.vm.provider "virtualbox" do |v|
             v.name = mastername
-            v.memory = 2048
+            v.memory = 6144
             v.cpus = 2
         end
         master.vm.hostname = mastername
-        master.vm.network "public_network", bridge: 'en0: Wi-Fi (AirPort)'
+        master.vm.network "private_network", ip: @priv_net_ip
```

Comment:
I'm eager to switch to private network, but with dhcp as default. We should do something like priv_net_ip || dhcp..